src/containers/save/save.container.js

```diff
@@ -0,0 +1,121 @@
+                                                item.style.transform
+                                            }%) translateZ(0)`,
+                                            transformStyle: 'preserve-3d',
+                                            opacity: item.style.opacity
+                                        }}>
+                                        <Toolbar
+                                            tabId={this.props.tab_id}
+                                            setDropDownStatus={
+                                                this.props.setDropDownStatus
+                                            }
+                                            openPocket={this.props.openPocket}
+                                            openOptions={this.props.openOptions}
+                                            archive={this.props.archiveItem}
+                                            remove={this.props.removeItem}
+                                            currentTab={this.props.currentTab}
+                                            active={this.props.active}
+                                            tags={this.props.currentTags}
+                                            activateTag={this.props.activateTag}
+                                            deactivateTag={
+                                                this.props.deactivateTag
+                                            }
+                                            deactivateTags={
+                                                this.props.deactivateTags
+                                            }
+                                            addTag={this.props.addTag}
+                                            closePanel={this.props.closePanel}
+                                            removeTag={this.props.removeTag}
+                                            removeTags={this.props.removeTags}
+                                            storedTags={
+                                                this.props.setup.tags_stored
+                                            }
+                                            inputFocused={
+                                                this.props.inputFocused
+                                            }
+                                            setInputFocusState={
+                                                this.props.setInputFocusState
+                                            }
+                                        />
+                                        {this.props.showRecs && (
+                                            <Recommendations
+                                                tabId={this.props.tab_id}
+                                                recs={this.props.currentRecs}
+                                                saveRecommendation={
+                                                    this.props
```

Comment:
We'll have to update the autoformatter because this specific line break is inconsistent with others on the page.

Reply:
I agree.  Consistency is a bit of a problem with the formater.  I think this is a twofold issue.